index.js

```diff
@@ -0,0 +1,45 @@
+
+
+
+const operations = ['+', '-', '/', '*']
+let total = 0
+
+const calculate = (calculationSteps) => {
+  if (operations.some(element => calculationSteps.includes(element)) === false) {
+    return NaN
+  }
+
+  calculationSteps = calculationSteps.filter(element => element != '')
+  for (let i = 0; i < calculationSteps.length; i++) {
+    if (calculationSteps[i] === null) {
+      calculationSteps[i] = 0
+    }
```

Comment:
You actually don't need this, `Number(null)` returns `0` so you could just use that as your typecast below and it would handle this case for you.